tests/features/change_tracking_workflow.feature

```diff
@@ -111,12 +127,19 @@ Feature: Change tracking features
       | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> | <p>No ice-ins or ice-del tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p> |
       | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    | <p>No tracking change <a href=\"http://www.europa.eu/newsroom\">The latest news</a></p>                    |
 
+  @api
   Scenario Outline: The change of the content state to "validated" or "published" must be blocked if
   CKEditor Lite tracked changes exist in WYSIWYG fields of a translation
-    Given the following languages are available:
+    Given I am logged in as a user with the 'administrator' role
+    And the following languages are available:
       | languages |
       | en        |
       | fr        |
+    When I go to "admin/config/content/multisite_wysiwyg/workbench"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-validated"
+    And I check the box "edit-nexteuropa-editorial-tracking-wbm-states-published"
+    And I press "Save configuration"
+    Then I should see the success message "The configuration options have been saved."
     When I am viewing a multilingual "page" content:
```

Comment:
I'm not sure I understand why this changes are made in this pull request. They don't seem to belong to this ticket at all?